src/main/java/io/fabric8/elasticsearch/plugin/kibana/KibanaSeed.java

```diff
@@ -99,6 +99,9 @@ public void setDashboards(final OpenshiftRequestContext context, Client client,
         LOGGER.debug("Found '{}' Index patterns for user", indexPatterns.size());
 
         Set<String> projects = new HashSet<>(context.getProjects());
+        if(context.isOperationsUser()) {
+            projects.add(OPERATIONS_PROJECT);
+        }
```

Comment:
Ah, so move it here, so it will get filtered out of the list if there is no `.operations.*` indices?  Ok

Reply:
Correct.  Assume there are indices and then when we check for their existence it will be filtered out of the set